data/map.txt

```diff
@@ -32310,6 +32315,7 @@ planet Zug
 	spaceport `The spaceport of Zug has been growing by amalgamation for centuries. At its core is the "Old Port," a set of quaint and rudimentary shops and warehouses. The landing pads for the Old Port were torn up to make room for what the locals call "The Port," a set of buildings that were constructed a century ago, with bolder architecture and more modern design.`
 	spaceport `	Some distance away is the "New Port," a towering structure with robotic cargo lifts and large hangars for more expensive ships. Meanwhile, plans are apparently underway for an expansion of the New Port. You can't help but wonder what the new expansion will be named.`
 	shipyard "Basic Ships"
+	shipyard "Tarazed Basics"
```

Comment:
```suggestion
	shipyard "Tarazed Basics"
	shipyard "Tarazed Advanced"
```

Reply:
Just a thought here: Instead of having Zug just simply "have" the advanced starport, it might make a lot more sense to have an event - perhaps with a short mission or two - in the very early FW campaign that completes that expansion of the New Port and adds the Tarazed Advanced shipyard. In terms of gameplay, it would add a small changing detail to the world and potentially let the player see the world changing around them, which tends to make things more immersive.

Reply:
Yea, that sounds like a good idea as well.

Probably best to do it in a separate PR, though, even if @bene-dictator wants to just knock that out anyway, no reason to hold up these other changes for it. :)

Reply:
I see your point with Zug, and once I figure out missions I'll give it a go. However, I never really knew why Solace never had a shipyard. It's an advanced planet with 2 billion inhabitants. Delta V Corporation would need a local shipyard to test maneuverability (or whatever) on ship models, and they presumably export their engines from Solace too, meaning that they'd need a local base for freighters and warships in case pirates from Thule attack them. I don't know why Glaze, with 270 million inhabitants but no real need or reason for a shipyard, has one, but I didn't change that.

Reply:
The Syndicate Diplomacy missions would be a good place to begin selling advanced Tarazed ships on FW worlds, as that same mission string already has outfitters from Tarazed moving to FW worlds.